tests/Jaeger/Codec/ZipkinCodecTest.php

```diff
@@ -40,7 +40,7 @@ function testInject()
         $this->assertEquals('7b', $carrier['X-B3-TraceId']);
         $this->assertEquals('1c8', $carrier['X-B3-SpanId']);
         $this->assertEquals('315', $carrier['X-B3-ParentSpanId']);
-        $this->assertEquals('1', $carrier['X-B3-Flags']);
+        $this->assertSame(1, $carrier['X-B3-Flags']);
```

Comment:
This `$carrier['X-B3-Flags']` will be `1` and it should use the `assertSame` to assert equals checking strictly.